lib/agentless/agentless.go

```diff
@@ -57,14 +68,14 @@ func SignerFromAuthzContext(authzCtx *authz.Context, generator CertGenerator) fu
 		identity := authzCtx.Identity.GetIdentity()
 		ttl := time.Until(identity.Expires)
 
-		signer, err := createAuthSigner(ctx, authzCtx.User.GetName(), identity.TeleportCluster, ttl, generator)
+		signer, err := createAuthSigner(ctx, authzCtx.User.GetName(), identity.TeleportCluster, ttl, identity.Groups, identity.Traits, generator)
 		return signer, trace.Wrap(err)
 	}
 }
 
 // createAuthSigner creates a [ssh.Signer] that is signed with
 // OpenSSH CA and can be used to authenticate to agentless nodes.
-func createAuthSigner(ctx context.Context, username, clusterName string, ttl time.Duration, generator CertGenerator) (ssh.Signer, error) {
+func createAuthSigner(ctx context.Context, username, clusterName string, ttl time.Duration, roles []string, traits wrappers.Traits, generator CertGenerator) (ssh.Signer, error) {
```

Comment:
I'd consider creating struct to pass arguments here as list gets pretty long

Reply:
I'll probably add one if it gets any longer, I think it's fine for now as it's not being used anywhere else